core/src/commonMain/kotlin/HexString.kt

```diff
@@ -0,0 +1,23 @@
+package com.juul.kable
+
+/**
+ * Copied from Tuulbox.
+ * https://github.com/JuulLabs/tuulbox/blob/fde4eb74d2aeb37b6aaac2002bccc553adc02c5a/encoding/src/commonMain/kotlin/HexString.kt
```

Comment:
Why not just include the dependency? `encoding` is a small artifact, and Tuulbox 4.3.0 added support for all of the targets we use.

Reply:
Good call out. At the time I copied this, I think I was blocked by missing some targets in `encoding` that Kable needed. Perhaps that was resolved with https://github.com/JuulLabs/tuulbox/pull/68. I'll give it another try.